bowtie2/2.4.4/Dockerfile

```diff
@@ -0,0 +1,62 @@
+# FROM defines the base docker image to start from. This command has to come first in the file
+FROM ubuntu:xenial
+
+ARG SAMTOOLSVER=1.11
+ARG BOWTIE2VER=2.4.4
+
+# metadata (there are a few other labels you can add, these are optional but preferred!)
+LABEL base.image="ubuntu:xenial"
+LABEL version="1"
```

Comment:
please change this line `LABEL version="1"` --> `LABEL dockerfile.version="1"` for clarity. See explanation here: https://github.com/StaPH-B/docker-builds/issues/118